external/CMakeLists.txt

```diff
@@ -64,8 +64,15 @@ if (VERONA_DOWNLOAD_LLVM)
   )
 else()
   separate_arguments(LLVM_EXTRA_CMAKE_ARGS UNIX_COMMAND ${LLVM_EXTRA_CMAKE_ARGS})
-  list (APPEND LLVM_EXTRA_CMAKE_ARGS
+  set (LLVM_CMAKE_ARGS
+    # set a default CMAKE_BUILD_TYPE to be the same as for Verona
     -DCMAKE_BUILD_TYPE=${CMAKE_BUILD_TYPE}
+
+    # append LLVM_EXTRA_CMAKE_ARGS (which may override CMAKE_BUILD_TYPE)
+    ${LLVM_EXTRA_CMAKE_ARGS}

```

Comment:
Move this to the end of the list, so that it overrides all args, not just build type

Reply:
moved